src/debug/debugSession.ts

```diff
@@ -361,10 +363,7 @@ export class DebugSession implements IDebugSession {
             if (proxyResult && proxyResult.proxyProcess) {

```

Comment:
I left this part in the default callback since it'll be needed at the end of every debug session to close port-forward. Once complete, I call the caller-specified callback for running and context-specific cleanup